Running in wrapper mode for '145'

Running python job.

[name: "CPU-QUOTA"
int_val: 100
int_val: 76
int_val: 85
int_val: 17
int_val: 99
int_val: 61
int_val: 98
, name: "DISK"
int_val: 10
int_val: 33
int_val: 40
int_val: 33
int_val: 8
int_val: 14
int_val: 5
, name: "MEMORY"
int_val: 13
int_val: 13
int_val: 38
int_val: 11
int_val: 23
int_val: 34
int_val: 12
, name: "NET"
int_val: 17
int_val: 26
int_val: 17
int_val: 21
int_val: 35
int_val: 33
int_val: 16
]
The paramters are {u'CPU-QUOTA': array([100,  76,  85,  17,  99,  61,  98]), u'NET': array([17, 26, 17, 21, 35, 33, 16]), u'DISK': array([10, 33, 40, 33,  8, 14,  5]), u'MEMORY': array([13, 13, 38, 11, 23, 34, 12])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
----------------------------------------
Problem running the job:
(<class 'paramiko.ssh_exception.BadAuthenticationType'>, BadAuthenticationType('Bad authentication type', [u'publickey']), <traceback object at 0x7f244e24e878>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 147, in explore_spearmint
    client = re.get_client(machine)
  File "/home/ubuntu/throttlebot/src/remote_execution.py", line 6, in get_client
    client.connect(ip, username="quilt", password="")
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 424, in connect
    passphrase,
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 714, in _auth
    raise saved_exception
BadAuthenticationType: ('Bad authentication type', [u'publickey']) (allowed_types=[u'publickey'])
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 0.43 seconds.
